test/interop/C/exportArray/returningArr/returnBadArray.chpl

```diff
@@ -1,6 +1,6 @@
 // This tests that it is an error to return an array with a different lower
 // bounds than expected
 export proc foo(): [] int {
-  var x = [1, 2, 3, 4]; // lower bounds is 1 (because that's the default)
+  var x: [1..4] int = [1, 2, 3, 4]; // lower bounds is 1
```

Comment:
It looks like there's another test in the same directory, returnBadArray2.chpl, that does a similar thing (but uses an explicit domain bounds in the return type).  I'm surprised to find it not included, did you forget to commit it?  The C test that relies on it shouldn't have been skipped so far as I can tell.

That was the only thing that concerned me in my assigned tests

Reply:
I believe that since `returnBadArray2.chpl` explicitly declares the domain value of the array being returned, there isn't a need to give explicit bounds to `x` because it'll be converted from a `0..3` array to a `1..4` array upon being returned.  This is similar to how the initialization just above is OK even though the arrays have different domains (since we only require shape matches for rectangular arrays).  Imagine that the routine in `returnBadArray2.chpl` being converted to something like:

```chapel
var retval: [1..4] int;
retval = x;
return retval;
```

whereas in returnBadArray.chpl, it's more like:

```chapel
var retval: [x.domain] int = x;
return retval;
```

and inherits x's domain which would be `{0..3}` if we didn't declare it explicitly in one place or the other.

Reply:
That's fair.  But looking at the file, the comment should still be updated to reflect the new normal

Reply:
https://github.com/chapel-lang/chapel/pull/14521/commits/e54c0bac53fddc065187259a7b400a63309b3f59 does this.

Reply:
Awesome, thanks!